src/test/v2.0/advanced/fdc3.open.ts

```diff
@@ -0,0 +1,368 @@
+// import { OpenError, Context, Channel, Listener, getOrCreateChannel } from "fdc3_1_2";
```

Comment:
Forgot to uncomment these?

Reply:
no.  We're waiting for Oli's merge, aren't we?

Reply:
this is a copy of the 1.2 tests atm